nearcore/src/lib.rs

```diff
@@ -401,6 +405,7 @@ pub fn start_with_config_and_synchronization(
         partial_witness_actor.clone().with_auto_span_context().into_multi_sender(),
         true,
         None,
+        resharding_sender.into_multi_sender(),
```

Comment:
Ugh, we need a better way to handle adding new actors and new actix requests. Note to self; I'll see if I can come up with better ways of handling all actor senders in a single unified struct.